types/unified-tests.ts

```diff
@@ -155,19 +155,19 @@ processor.use(
     (
       tree: Node,
       file: VFile,
-      next: (error: Error | null, tree: Node, file: VFile) => {}
+      next: (error: Error | null, tree: Node, file: VFile) => void
     ) => {}
 )
 processor.use(
   () =>
     (
       tree: Node,
       file: VFile,
-      next: (error: Error | null, tree: Node) => {}
+      next: (error: Error | null, tree: Node) => void
     ) => {}
 )
 processor.use(
-  () => (tree: Node, file: VFile, next: (error: Error | null) => {}) => {}
+  () => (tree: Node, file: VFile, next: (error: Error | null) => void) => {}
```

Comment:
could we add a test case where next is called without any params to prevent a regression?

Reply:
done!